_sass/main.scss

```diff
@@ -9,3 +9,40 @@
 
 @import 'patterns_syntax-highlight';
 @include vfio-p-syntax-highlight;
+
+.p-thin-font {
```

Comment:
Agree we need this but think it's better named after its function not style. For example, `p-heading--stylized`